src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -1043,6 +1043,19 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                             // So set it (this is what the symsrv code on Windows sets).   On NetStandard1.6 we give up since we dont' have this API available.  
                             req.UserAgent = "Microsoft-Symbol-Server/6.13.0009.1140";
 #endif
+
+#if NET462
```

Comment:
NET462 makes this work for PerfView, but other TraceEvent consumers that run against .NET Core can't use it.  The identity packages appear to support various netstandard versions.  Any reason we can't relax this?